### PR TITLE
Added location filter to pnlppgi reports

### PR DESCRIPTION
**Pull request by lwyszomi**
@NoahCarnahan

### CODE REVIEW

File: custom/pnlppgi/filters.py [hunk 1]
```diff
@@ -16,3 +18,60 @@ def selected(self):
...
+
+
+class LocationBaseDrilldownOptionFilter(BaseDrilldownOptionFilter):
+    slug = 'id'
+    label = 'Location'
+
+    @classmethod
+    def get_labels(cls):
+        return [
+                ('Zone', 'Select Zone', 'zone'),
```
Comment (dimagimon) [inline custom/pnlppgi/filters.py]: E126 continuation line over-indented for hanging indent

File: custom/pnlppgi/filters.py [hunk 2]
```diff
@@ -16,3 +18,60 @@ def selected(self):
...
+                ('Region', 'Select Region', 'region'),
+                ('District', 'Select Form', 'district'),
+                ('Site', 'Select Site', 'site')
+            ]
```
Comment (dimagimon) [inline custom/pnlppgi/filters.py]: E121 continuation line under-indented for hanging indent

File: custom/pnlppgi/utils.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
+from couchdbkit import ResourceNotFound
+from sqlagg.filters import EQ
+
+from corehq.apps.groups.models import Group
+from corehq.apps.locations.models import SQLLocation
```
Comment (dimagimon) [inline custom/pnlppgi/utils.py]: F401 'SQLLocation' imported but unused

File: custom/pnlppgi/filters.py [hunk 4]
```diff
@@ -16,3 +18,60 @@ def selected(self):
...
+class LocationBaseDrilldownOptionFilter(BaseDrilldownOptionFilter):
+    slug = 'id'
+    label = 'Location'
+
+    @classmethod
+    def get_labels(cls):
+        return [
+            ('Zone', 'Select Zone', 'zone'),
+            ('Region', 'Select Region', 'region'),
+            ('District', 'Select Form', 'district'),
```
Comment (NoahCarnahan) [inline custom/pnlppgi/filters.py]: Looks like this should say "Select District", not "Select Form".

**Comment by NoahCarnahan** [comment]
Looks good, thanks.
Merging now so that this will hopefully be deployed today, but you should open another PR to fix the "Select Form" string (mentioned in comment).

**Pull request merged**